rest_framework-stubs/schemas/__init__.pyi

```diff
@@ -18,7 +18,7 @@ def get_schema_view(
     renderer_classes: Optional[Sequence[Type[BaseRenderer]]] = ...,
     public: bool = ...,
     patterns: Optional[Sequence[Any]] = ...,
-    generator_class: Type[SchemaGenerator] = ...,
+    generator_class: Type[Union[openapi.SchemaGenerator, coreapi.SchemaGenerator]] = ...,
```

Comment:
Maybe we can just go with `BaseSchemaGenerator`? https://github.com/encode/django-rest-framework/blob/0323d6f8955f987771269506ca5da461e2e7a248/rest_framework/schemas/generators.py#L146

Reply:
👍